taqueria-vscode-extension-web-ui/src/data-editors/OptionEditor.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { DataEditorNode } from './DataEditorNode';
+import { VSCodeCheckbox } from './VsCodeWebViewUIToolkitWrappers';
+
+export const OptionEditor = (
+	{ dataType, value, onChange }: { dataType: any; value: any; onChange: (value: any) => void },
+) => {
+	const changeValue = (v: any) => {
```

Comment:
Is it possible to avoid the `any` type here?

Reply:
Yes, Done